bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/profiles/Profile.java

```diff
@@ -19,6 +22,16 @@
  * @author Simon Kaufmann - initial contribution and API.
  *
  */
+@NonNullByDefault
 public interface Profile {
 
+    ProfileTypeUID getProfileTypeUID();
```

Comment:
This should probably be static. It was annoying to me that it wasnt static when deriving from it.